functional-tests/affected-ftest/pom.xml

```diff
@@ -42,6 +42,10 @@
           <properties>
             <orderStrategy>affected</orderStrategy>
```

Comment:
if we introduce two approaches how to apply the strategies:
* ordering
* filtering 
then the name `orderStrategy` can be quite confusing for the filter approach... I would change it to just `strategy` or `strategies`